frontend/packages/console-shared/src/modals/modal-resource-launcher.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as _ from 'lodash';
+import { Provider } from 'react-redux';
+import { Router } from 'react-router-dom';
+
+import store from '@console/internal/redux';
+import { Firehose, AccessDenied, history, MsgBox, Box } from '@console/internal/components/utils';
+import { ModalErrorContent } from '@console/internal/components/modals/error-modal';
+
+const NotFound = ({ message }) => (
+  <Box className="text-center">
+    <MsgBox title="Not Found" detail="Requested resource not found." />
+    {_.isString(message) && (
+      <div className="alert alert-danger text-left">
```

Comment:
We're moving to PF4 alerts, although an inline alert in a dialog seems strange. Is this following a design?

Reply:
Not this one. Designers approved the notification used in `web-ui` but I had to change it for openshift/console to meet requirements about code-base so far discussed in other PRs.

If a resource-retrievel error occurs while opening the modal dialog in the `web-ui`, it will be closed and a new modal dialog rendering the error message will be displayed with 1 second delay instead.

Implementation within this PR does something similar, but reflects that the Firehose's children are not rendered until data retrieval is finished. So I can render an error-modal immediately.

The `<NotFound />` implemented here just reuses design from `<StatusBox />`. I have no issue in changing that to something better.